internal/cmd/kafka/command_cluster_update.go

```diff
@@ -43,7 +43,7 @@ func (c *clusterCommand) newUpdateCommand(cfg *config.Config) *cobra.Command {
 }
 
 func (c *clusterCommand) update(cmd *cobra.Command, args []string) error {
-	if !cmd.Flags().Changed("name") && !cmd.Flags().Changed("cku") {
+	if err := errors.CheckNoOpUpdate(cmd.Flags(), "cku", "name"); err != nil {
 		return errors.New(errors.NameOrCKUFlagErrorMsg)
```

Comment:
```suggestion
		return err
```

Reply:
It looks like this one was missed when replacing the error messages. Since this looks like the only usage of this string, we should be good to remove it from `error_message.go` as well.